podcasts/Chapters.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Chapters.swift
+//  podcasts
+//
+//  Created by Maarut Chandegra on 11/11/2022.
+//  Copyright © 2022 Shifty Jelly. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Chapters: Equatable {
+    private var chapters = [ChapterInfo]()
+
+    init(chapters: [ChapterInfo]) {
```

Comment:
I see elsewhere in the code you're init'ing with an empty array. I think we can clean up the code by adding a default value, that way you can just do `Chapters()`

```suggestion
    init(chapters: [ChapterInfo] = []) {
```

Reply:
Default value defined on init.